src/edgepi/dac/README.md

```diff
@@ -6,17 +6,24 @@ This section will demonstrate how to import the EdgePi DAC module, and use it wr
 Note, the EdgePi has eight analog out pins, indexed from 1 to 8.
 
 ### Writing Voltage to Analog Out Pin
-```
+```python
+from edgepi.dac.dac_constants import DACChannel as Ch
 from edgepi.dac.edgepi_dac import EdgePiDAC
 
 # initialize DAC
 edgepi_dac = EdgePiDAC()
 
+# setting DAC range 0-5V
+edgepi_dac.enable_dac_gain(False)
```

Comment:
We still need to call `enable_dac_gain` right after initializing `EdgePiDAC`, right? If so, does this really close #290?

What if there is a scenario where we want to enable (or disable) DAC gain and we have a `EdgePiDAC` object already instantiated? Would we have to instantiate another `EdgePiADC` object?

Reply:
I should update the Readme doc. 

Yes, you still have to call enable_dac_gian() when you instantiate the object, but it doesn't require instantiating another EdgePiDAC object. It is only required to be called only once.

enable_dac_gain() has another flag which by default is False. When True is passed, it will update channel values to keep the current voltage.